content/blog/2026-06-24-secret-life-of-data/index.md

```
@@ -0,0 +1,124 @@
+
+|          | Cluster size | gb wrong-sized  | Wrong-sized key size            | Primary size | # of Primaries |
+| -------- | ------------ | --------------- | ------------------------------- | ------------ | -------------- |
+| Original | 100gb        | 95gb            |  212                            | 20gb        | 5              |
+| New      | 58.58gb      | 0               |  120 <br /> (56.6% of original) | 20gb        | 3              |
+
+So, you can take away two full primaries and whatever associated replicas. As previously mentioned, this is an extreme example, it's unlikely that you have 95% of your data falling outside this threshold. Let's say that you have 60% of your keys that reach above configuration threshold _and_ you can scale back your infrastructure to a smaller-sized machine or instance that has only 16gb of RAM.
+
+|          | Cluster Size | GB Above Threshold | Memory Usage per Key    | Primary Node Size | # of Primaries |
+| -------- | ------------ | ------------------ | ------------------------------- | ----------------- | -------------- |
+| Original | 100 GB       | 60 GB              |  212 B                            | 20 GB             | 5              |
+| New      | 74 GB        | 0                  |  120 B <br /> (56.6% of original) | 14.8 GB           | 5              |```
```

markdownlint-cli2 (0.23.0)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)